src/pages/Home/TodoModel/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React, { useRef } from "react";
+import { useDispatch } from "react-redux";
+import { addTask } from "../../../store/actions";
+import { v1 as uuidv1 } from "uuid";
+import Todos from "../Todos";
+import style from './index.module.scss'
+const TodoModel = () => {
+  const refTask = useRef("");
+  const dispatch = useDispatch();
+  const onSubmit = (e) => {
+
+    if (e.key === "Enter") {
+      e.preventDefault();
+
+      if(!refTask.current.value){
+        return
+      }
+      
+      let task = {
+        id: uuidv1(),
+        task: refTask.current.value,
```

Comment:
Why are we using reference to get the value? We can use `useState` and `onChange` method of capture the value